Fixed RemovedBasePackages list's lack of wrapped output values

duke7553 opened the pull request:
Hello everybody,

My last experience submitting a PR and addressing your feedback with it really satisfied me, so I'm opening another one here!

A regression was reported with the changes made to the status output of RemovedBasePackages to group EVRs, which caused the output to no longer wrap "nicely."

This PR fixes that by adding a new built-in method to print key-value pairs in a way that retrieves the respective comma-separated values directly from a string. It does this in a way that accounts for the potential width of the text area.

This might be overkill, but we'll see. I'm very excited to hear your feedback.

Fixes #1613 

As you can see, the screenshot below depicts the output after & before my changes: 
![Screenshot from 2020-05-17 01-18-40](https://user-images.githubusercontent.com/20365014/82136735-9bc2f300-97de-11ea-9c93-789a2f571452.png)

duke7553 commented:
/assign @miabbott

miabbott commented:
/ok-to-test

duke7553 commented:
@miabbott It doesn't look like the ok-to-test label changed on the PR

miabbott commented:
:thinking: 

/ok-to-test
/retest

miabbott commented:
@duke7553 Not sure what is going on with the robots here

I did notice that your PR bumps the `libdnf` and `libglnx` submodules...is that intended and/or required for your PR?

duke7553 commented:
@miabbott No it's not intended. That's weird. I'll have to fix that part

duke7553 commented:
@miabbott I removed the accidental submodule changes.

miabbott commented:
@duke7553 Now it looks like some Rust crates got bumped and a commit from @jlebon got swallowed by your own?

Also, the CI appears to have caught an error in your change itself.

duke7553 commented:
> Now it looks like some Rust crates got bumped and a commit from @jlebon got swallowed by your own?

There's a good chance I squashed the commits incorrectly, so I reverted that for now.

jlebon commented:
@duke7553 Did you want to pick this up again?

duke7553 commented:
@jlebon School has gotten busy again, so I can't at the moment. Sorry about that.
